sdks/python/container/build.gradle

```diff
@@ -42,8 +42,33 @@ tasks.register("buildAll") {
   dependsOn ':sdks:python:container:py311:docker'
 }
 
+for(int i=8; i<=11; ++i) {
+  String cur = "3" + i
+  String prev = "3" + (i-1)
+  tasks.register("push" + cur) {
+    if (cur != "38") {
```

Comment:
ok, it's the first one in the chain

Reply:
we should try to hardcode python versions in as few pieces as possible. Ideally, we should have lowest supported and highest supported constant, similar to https://github.com/apache/beam/blob/42ca3bf598e70e3baa550859de1d2ca6712bd8f6/.test-infra/jenkins/PythonTestProperties.groovy#L31 . Maybe in top level settings.gradle. Other place that would benefit from this: https://github.com/apache/beam/blob/master/sdks/python/test-suites/gradle.properties . If that's too much hassle right now, consider at least not adding lowest/highest constant in this file, so that we programmatically evaluate line 49.

Reply:
I extracted to variable constants here to keep the change small since there's a good chance we're cherry picking this